website/content/docs/connect/dataplane/telemetry.mdx

```diff
@@ -0,0 +1,42 @@
+---
+layout: docs
+page_title: Consul Dataplane - Enable Telemetry Metrics
+description: >-
+  Configure telemetry to collect metrics you can use to debug and observe Consul Dataplane behavior and performance.
+---
+
+# Consul Dataplane Telemetry
+
+Consul Dataplane collects metrics about its own status and performance.
+The following external metrics stores are supported:
+
+- [DogstatsD](https://docs.datadoghq.com/developers/dogstatsd/)
+- [Prometheus](https://prometheus.io/docs/prometheus/latest/)
+- [StatsD](https://github.com/statsd/statsd)
+
+Consul Dataplane respects the telemetry settings used for Envoy. The external metrics store can be
+specified in the `proxy-defaults` configuration entry or directly in the `proxy.config` field of the
+proxy service definition. Refer to the [Envoy bootstrap
+configuration](/docs/connect/proxies/envoy#bootstrap-configuration) for details.
```

Comment:
Yeah, this could be more precise.

> When we say that "Consul Dataplane respects the telemetry settings used for Envoy", do we mean that CD supports Envoy's telemetry settings?

Right. The specific settings listed [in the Envoy bootstrap configuration section](https://developer.hashicorp.com/consul/docs/connect/proxies/envoy#bootstrap-configuration) are used to configure telemetry for _both_ Envoy and Consul Dataplane.

> Can we say "Consul Dataplane telemetry settings do not interfere with Envoy's telemetry settings" ?

I think this is accurate, but it's more precise to say that Envoy's telemetry settings are also used to configure Consul Dataplane telemetry.

Here's another take at phrasing this: 

"Consul Dataplane uses the same external metrics store that is configured for Envoy. To enable telemetry for Consul Dataplane, enable telemetry for Envoy by specifying an external metrics store in the `proxy-defaults` configuration entry or directly in the `proxy.config` field of the proxy service definition."

Let me know what you think!

Reply:
> Yeah, this could be more precise.
> 
> > When we say that "Consul Dataplane respects the telemetry settings used for Envoy", do we mean that CD supports Envoy's telemetry settings?
> 
> Right. The specific settings listed [in the Envoy bootstrap configuration section](https://developer.hashicorp.com/consul/docs/connect/proxies/envoy#bootstrap-configuration) are used to configure telemetry for _both_ Envoy and Consul Dataplane.
> 
> > Can we say "Consul Dataplane telemetry settings do not interfere with Envoy's telemetry settings" ?
> 
> I think this is accurate, but it's more precise to say that Envoy's telemetry settings are also used to configure Consul Dataplane telemetry.
> 
> Here's another take at phrasing this:
> 
> "Consul Dataplane uses the same external metrics store that is configured for Envoy. To enable telemetry for Consul Dataplane, enable telemetry for Envoy by specifying an external metrics store in the `proxy-defaults` configuration entry or directly in the `proxy.config` field of the proxy service definition."
> 
> Let me know what you think!

👍🏽